.gitignore

```diff
@@ -0,0 +1 @@
+forger
```

Comment:
This is just some dummy file... if you have python scripts that makes multiple cards (for phase space cut variations), place your scripts in `forger` directory